docs/source/using/experimental-features.adoc

```diff
@@ -152,6 +152,170 @@ To get the status of the DNS server:
+
+=== Adding the Network Service to _ServiceOrder_ array
+
+In the *_/Library/Preferences/SystemConfiguration/preferences.plist_* file, look for the `<key>ServiceOrder</key>` element.
+As root, append the UUID for our *MiniTap* network service to this array. 
+
+----
+<key>ServiceOrder</key>
+    <array>
+        <string>06BFF3C7-13DA-420F-AE9C-B036401184D7</string>
+	<string>58231F56-CA25-4D41-930F-46D83CA07BFE</string>
+	<string>304203B0-AC87-459F-9761-C2799EEBB2E3</string>
+	<string>8655D244-C6E7-4CC0-BF06-BB18F9C3BB85</string>
+	<string>3C26FB9D-D918-4B79-9C7B-ADECD8EFE00F</string>
+	<string>D16F22CE-6DDE-4E63-837C-E16538EA5CCB</string>	//<1>
+    </array>
+----
+
+<1> The UUID for *MiniTap* network service.
+
+=== Adding the Network Service to _Service_ dictionary
+
+In the *_/Library/Preferences/SystemConfiguration/preferences.plist_* file, look for the `<key>Service</key>` element.
+As root, append the following XML to its dictionary:
+
+----
+<key>Service</key>
+    <dict>
+        <key>06BFF3C7-13DA-420F-AE9C-B036401184D7</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/06BFF3C7-13DA-420F-AE9C-B036401184D7</string>
+        </dict>
+        <key>304203B0-AC87-459F-9761-C2799EEBB2E3</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/304203B0-AC87-459F-9761-C2799EEBB2E3</string>
+        </dict>
+        <key>3C26FB9D-D918-4B79-9C7B-ADECD8EFE00F</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/3C26FB9D-D918-4B79-9C7B-ADECD8EFE00F</string>
+        </dict>
+        <key>58231F56-CA25-4D41-930F-46D83CA07BFE</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/58231F56-CA25-4D41-930F-46D83CA07BFE</string>
+        </dict>
+        <key>8655D244-C6E7-4CC0-BF06-BB18F9C3BB85</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/8655D244-C6E7-4CC0-BF06-BB18F9C3BB85</string>
+        </dict>
+        <key>D16F22CE-6DDE-4E63-837C-E16538EA5CCB</key>					//<1>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/D16F22CE-6DDE-4E63-837C-E16538EA5CCB</string>	//<2>
+        </dict>
+    </dict>
+----
+
+<1> The UUID of the *MiniTap* service.
+<2> Replace this UUID with the UUID of your *MiniTap* service.
+
+Reboot macOS and you should see a *MiniTap* service in the Network Preferences GUI.
+This service will be disconnected.
+To turn it on, issue the following commands:
+
+----
+exec 4<>/dev/tap0			//<1>
+ifconfig tap0 10.10.90.1 255.255.0.0    //<1> <2>
+ifconfig tap0 up			//<1>
+----
+
+<1> Replace it with the `/dev/tap` device used by *MiniTap* Service.
+<2> IP address should be same as the one in the *MiniTap* Service definition.
+
+=== Adding resolver config
+
+Create the file `/etc/resolver/nip.io` with the following content:
+
+----
+nameserver <ip_address_of_the_minishfit_vm>
+search_order 1
+----
+
```

Comment:
@anjannath @gbraad  There are lot of steps for this work. We can add this documentation but we need to start working towards automating these asap.

Reply:
do we want to include comments that are actually XML comments or do we want them to break the config?

Reply:
When the docs are rendered the finale output will not contain the call-out's original text like `//<1>`
Is this what you are referring to?

Reply:
@anjannath  What @gbraad meant is , whether user should be able to copy the XML file from the documentation ? If we want users to copy the comments then we need add comments as supported by XML.  @gbraad  right? See https://www.tutorialspoint.com/xml/xml_comments.htm

Though the current format looks good to eyes, we should only add XML comments, so that user can copy paste the XML.

Reply:
I am fine with current format as the user can copy it and use it. Check https://github.com/minishift/minishift/pull/2449/files#r195426685